src/components/CommentsList.jsx

```diff
@@ -0,0 +1,28 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { getComments } from "../utils/api";
+import CommentListing from "./CommentListing";
+
+const CommentsList = () => {
+  const [comments, setComments] = useState([]);
+  const { review_id } = useParams([]);
+  useEffect(() => {
```

Comment:
Could we have a loading state for comments? Also something to indicate that there are no comments rather than empty space

Reply:
Added loading state, thank you. I believe the implementation of task 8 (add a new comment to an existing review will serve this function for my page therefore I am tempted to just leave it empty for the time being as I struggling to think of a good in-between to implement.